src/app/(client)/onboarding/components/first-section.tsx

```
@@ -0,0 +1,58 @@
+import Image from 'next/image';
+import { useRouter } from 'next/navigation';
+import SITE_MAP from '@/constants/site-map.constant';
+import { useUserStore } from '@/store/user-store';
+
+const FirstSection = ({ setCookie }: { setCookie: () => void }) => {
+  const user = useUserStore((state) => state.user);
+  const router = useRouter();
+
+  const handleClickCtaButton = () => {
+    setCookie();
+    if (!!user.id) {
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)